llvm/test/Transforms/LoopVectorize/AArch64/prefer-fixed-if-equal-to-scalable.ll

```diff
@@ -0,0 +1,196 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
+; RUN: opt -S < %s -passes=loop-vectorize | FileCheck %s
```

Comment:
I wonder if this way of testing is potentially fragile due to future cost model changes. It might end up choosing the values you expect anyway regardless of your new feature. I've not tried this out, but can you use `-force-target-instruction-cost=1` to ensure you're testing the new code path?